bluebottle/bluebottle_dashboard/urls.py

```diff
@@ -0,0 +1,66 @@
+import django
```

Comment:
Why is this here? The monkey patching should be enough right? And why do we mark these urls as `csrf_exempt`?

Reply:
Remove this file. We are not using it

Reply:
We are not using this file. Please remove it.